ui-ngx/src/app/modules/home/components/event/event-table.component.ts

```diff
@@ -114,8 +128,18 @@ export class EventTableComponent implements OnInit, AfterViewInit, OnDestroy {
       this.debugEventTypes,
       this.overlay,
       this.viewContainerRef,
-      this.cd
+      this.cd,
+      this.nodeScriptTestService,
+      this.isRuleNodeDebugModeEnabled,
+      this.editingRuleNodeHasScript,
+      this.rulenodeClazz
     );
+
+    this.eventTableConfig.debugEventSelectedSubject.subscribe((debugEventBody: DebugRuleNodeEventBody) => {
+      if (debugEventBody) {
+        this.debugEventSelected.emit(debugEventBody);
+      }
+    })
```

Comment:
debugEventSelected parameter can be passed as an EventTableConfig class constructor parameter